### ISSUE TITLE
enable multiple indicators on same page

### ISSUE THREAD

**Issue by larham** [JakeWharton/ViewPagerIndicator#122]
such as a title strip on top, and dots on bottom.  currently, the last indicator "wins"--it shows, but first one does not.

**Comment by navarr** [comment]
This should already be possible.  In one of my apps I have three different indicators - unless you're referring to them both pointing to the same viewpager?

**Comment by larham** [comment]
thanks for clarifying.

yes, for the same viewpager, e.g., at bottom and top of display

**Comment by JakeWharton** [comment]
How do you propose this be done? The indicators currently subscribe to the page change events from `ViewPager` which only accepts a single listener at a time.

I can think of gross ways of doing this but I'm not going to implement them unless the 99.9999% case (single indicator per pager) is still easy.

**Comment by navarr** [comment]
This is probably one of the gross ways: but couldn't a pager subscribe to the page change events that another pager sends out?

**Comment by larham** [comment]
Yes, as navarr suggested, one way would be for the PageIndicator.setViewPager() to have an analogous setParentIndicator() if the parent PageIndicator had a setOnPageChangeListener().  

This is just a minor feature request. If the interfaces aren't sufficient, there isn't a justification for gross measures.

**Comment by JakeWharton** [comment]
Ideally we could change `ViewPager` to accept multiple observers. I may look into that first and contribute to AOSP for the next support library revision.

**Comment by iamyashrs** [comment]
Any updates regarding implementing this feature?

**Comment by AlmogBaku** [comment]
Hey, attached my solution here:
https://gist.github.com/AlmogBaku/7411ffa44030bc7c641d

This is a simple aggregator.. Nothing complex :)

**Comment by androidvsourz** [comment]
@AlmogBaku You saved me. Exactly what i want. Thumbs up /_/_/_/_/*
